pkg/apiserver/registry/stats/networkpolicystats/rest.go

```diff
@@ -65,8 +66,18 @@ func (r *REST) List(ctx context.Context, options *internalversion.ListOptions) (
 	if !features.DefaultFeatureGate.Enabled(features.NetworkPolicyStats) {
 		return nil, errors.NewBadRequest("feature NetworkPolicyStats disabled")
 	}
+	labelSelector := labels.Everything()
+	if options != nil && options.LabelSelector != nil {
+		labelSelector = options.LabelSelector
+	}
 	ns, _ := request.NamespaceFrom(ctx)
-	items := r.statsProvider.ListNetworkPolicyStats(ns)
+	stats := r.statsProvider.ListNetworkPolicyStats(ns)
```

Comment:
not sure if this is actually a good idea, but we could avoid the extra list copy by passing the label selector down to `ListNetworkPolicyStats`?

Reply:
I thought about this, however because we don't real use labels in statsProvider, I felt not worth to add something unrelated to the interface and the implementation to save a little overhead, and wanted to limit the labelSelector's scope to rest only. Does it make sense to you?

Reply:
makes sense